test/karma.conf.js

```diff
@@ -1,71 +1,28 @@
-module.exports = function(config) {
-  var customLaunchers = {
-    chrome_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'chrome',
-      platform: 'Windows 8.1',
-      version: '34'
-    },
-
-    firefox_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'firefox',
-      platform: 'Linux',
-      version: '29'
-    },
-
-    safari_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'safari',
-      platform: 'OS X 10.8'
-    },
-
-    ipad_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'ipad',
-      platform:'OS X 10.9',
-      version: '7.1'
-    },
-
-    android_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'android',
-      platform:'Linux'
-    },
-
-    ie_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'internet explorer',
-      platform: 'Windows 8.1',
-      version: '11'
-    }
-  };
+var customLaunchers;
 
+module.exports = function(config) {
   config.set({
     basePath: '',
 
-    frameworks: ['qunit'],
-
+    frameworks: ['browserify', 'qunit'],
     autoWatch: false,
-
     singleRun: true,
 
     files: [
-      // include and execute the standalone test bundle first
-      '../build/temp/tests.js',
-
-      // then include video.js through globals to run the API tests
-      '../build/temp/video-js.min.css',
-      '../build/temp/video.min.js',
-      'api/api.js'
+      '../build/temp/video-js.css',
+      '../test/globals-shim.js',
+      '../test/unit/**/*.js'
     ],
 
+    preprocessors: {
+      '../test/**/*.js': [ 'browserify' ]
```

Comment:
I haven't seen any problems with the browserify preprocessor yet. What trouble did you have?

And yeah, just remembered the third piece in all this is the tests' browser UI. Bah. That would either need to be a third watch task, or share the file with karma like we were doing. The biggest problem I saw with how we were doing it was the line numbers in errors were useless. With this change the line numbers aren't perfect but they're better. Any idea what we were doing wrong before to lose the source map chain?